avm/res/compute/virtual-machine-scale-set/tests/e2e/linux.defaults/main.test.bicep

```diff
@@ -0,0 +1,100 @@
+targetScope = 'subscription'
+
+metadata name = 'Using only defaults for Linux'
+metadata description = 'This instance deploys the module with the minimum set of required parameters.'
+
+// ========== //
+// Parameters //
+// ========== //
+
+@description('Optional. The name of the resource group to deploy for testing purposes.')
+@maxLength(90)
+param resourceGroupName string = 'dep-${namePrefix}-compute.virtualmachinescalesets-${serviceShort}-rg'
+
+@description('Optional. The location to deploy resources to.')
+param resourceLocation string = deployment().location
+
+@description('Optional. A short identifier for the kind of deployment. Should be kept short to not run into resource-name length-constraints.')
+param serviceShort string = 'cvmsslinmin'
+
+@description('Optional. Enable telemetry via a Globally Unique Identifier (GUID).')
+param enableTelemetry bool = true
```

Comment:
Should not be a parameter (nor used) in any AVM test file

Reply:
done